crates/oxide/src/parser.rs

```diff
@@ -1,1757 +0,0 @@
-use crate::{cursor::Cursor, fast_skip::fast_skip};
```

Comment:
I dropped this, but maybe we can keep it around somewhere for additional benchmarks?